otherlibs/win32unix/unixsupport.c

```diff
@@ -170,45 +170,118 @@ void win32_maperr(DWORD errcode)
 }
 
 /* Windows socket errors */
-
+#ifndef EWOULDBLOCK
```

Comment:
What happens if EWOULDBLOCK is already defined but is not equal to -WSAEWOULDBLOCK ?  Would that break the reporting of errors as a Unix_error exception?